src/styles/blocks/calendar.scss

```
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  max-width: $max-width;
+
+  gap: $gap-day;
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    min-width: $size-day-width;
+    min-height: $size-day-height;
+    font-family: $default-font;
+    font-size: $default-size;
+
+    box-sizing: border-box;
+    border: 1px solid black;
+    transition: transform 0.5s;
```

Usually you put the same value for transition duration on whole your project, so, make variable for it
```suggestion
    transition: transform $transitionDuration;
```